app/scripts/lib/getObjStructure.js

```diff
@@ -7,7 +7,7 @@ module.exports = getObjStructure
 
 // {
 //   "data": {
-//     "CurrencyController": {
+//     "CurrencyRatesController": {
```

Comment:
I assume this is part of a global search and replace, but in that case shouldn't be CurrencyRateController, without the **s**

Reply:
Great catch, this was a manual update. Fixed.